### PR TITLE
feat: add sitemap for better SEO

### PR DESCRIPTION
**Pull request by nawinsharma**
this pr solves #14

### CODE REVIEW

**Comment by nawinsharma** [comment]
@anisharaz  can you review it?

File: app/sitemap.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { MetadataRoute } from 'next';
+
+export default async function sitemap(): Promise<MetadataRoute.Sitemap> {
+  const baseUrl = 'https://blockx3.xyz';
+
+  const staticPages = ['', '/'].map((route) => ({
```
Comment (anisharaz) [inline app/sitemap.ts]: This will create array of 2 similar object , instead just use [""] as the array

**Comment by anisharaz** [comment]
there exist a rebots.txt file already , please remove that to add robots.ts

**Comment by nawinsharma** [comment]
I have added robots.ts file which is different

**Comment by nawinsharma** [comment]
@anisharaz  done with changes!

**Comment by anisharaz** [comment]
Thank you !! @NawinKumarSharma for the updates.

**Review by anisharaz — APPROVED** [review]

**Pull request merged**